rs-synthesis.Rmd

```diff
@@ -14,13 +14,23 @@ library(forcats)
 library(DT)
 library(metafor)
 
+theme_set(theme_bw())
```

Comment:
Wait, what?!? Are you giving up on `theme_minimal()`?

Reply:
Ugh, maybe I should stay true to myself. Or compromise with `theme_light()`